### PR TITLE
add install instructions to i3lock page

### PR DESCRIPTION
**Pull request by jtpereyda**
Add i3lock install instructions. As discussed in https://github.com/i3/i3lock/pull/223

### CODE REVIEW

File: i3lock/index.html [hunk 1]
```diff
@@ -32,6 +32,16 @@ <h2>Improvements</h2>
   </li>
 </ul>
 
+<h2>Install</h2>
+
+<p>
+  Many operating system i3 packages come with i3lock bundled (see
```
Comment (Airblader) [inline i3lock/index.html]: Can we rephrase this to

> Many distributions include i3lock as a (potentially optional) dependency of the i3 package (see Downloads). You can also look for the specific package in your distribution, e.g. on Debian and Debian-based distributions:

Comment (jtpereyda) [reply i3lock/index.html]: Thanks for the review! Updates pushed.

**Comment by Airblader** [comment]
Thank you!
